components/progress-bar/src/ProgressBar.js

```diff
@@ -0,0 +1,90 @@
+/** @jsx jsx */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme } from '@westpac/core';
+
+import { SrOnly } from '../../accessibility-helpers/src';
+
+// ==============================
+// Utils
+// ==============================
+
+// ==============================
+// Component
+// ==============================
+
+/**
+ * Progress Bar: A visual indication of progress. Use when loading content or to indicate how far along the user is in a journey.
+ */
+export const ProgressBar = ({ valuemin, valuemax, valuenow, skinny, ...props }) => {
+	const { progressBar } = useTheme();
+
+	// Common styling
+	const common = {
+		height: skinny ? 10 : 24,
+		marginBottom: progressBar.marginBottom,
+		overflow: 'hidden',
+		backgroundColor: 'white',
+		borderRadius: 1000,
+		border: `${progressBar.borderWidth} solid`,
+		borderColor: progressBar.borderColor,
+		position: 'relative',
```

Comment:
You're missing the clever little `::after` pseudo-element that Dom built into this 👀

Reply:
Nice one!